src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -31,15 +32,31 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
      * Returns true if none of the prefixes contains empty {@code Optional} values in the given
      * {@code ArgumentMultimap}.
      */
-    public static void assertPrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes)
-            throws MissingPrefixesException {
+    public static void assertAllPrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes)
+            throws AllPrefixesMissingException {
         Prefix[] missingPrefixes = Stream.of(prefixes)
                 .filter(prefix -> argumentMultimap.getValue(prefix).isEmpty())
                 .map(prefix -> new Prefix(prefix.getPrefix()))
                 .toArray(Prefix[]::new);
 
         if (missingPrefixes.length != 0) {
-            throw new MissingPrefixesException(missingPrefixes);
+            throw new AllPrefixesMissingException(missingPrefixes);
+        }
+    }
+
+    /**
+     * Returns true if at least one of the prefixes has non-empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    public static void assertAnyPrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes)
```

Comment:
I think you also need to check if the keyword is empty since when I typed `find n/` there is no error message. Similar to `find n/yu i/` or `find l/`,etc.

Reply:
Okay, will fix. Should also investigate other commands which allow for optional fields, since they also can be empty without raising an error (prefix present and prefix is "" is different)

```add student n/HZ i/A0000000B p/12345678 e/test@test h/hi m/CS1101 l/W1 /t```